src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -1380,6 +1384,69 @@ GUI_TEST_CLASS_DEFINITION(test_7451) {
     GTUtilsStartPage::checkRecentListUrl(os, "test_7451.fa", false);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7455) {
+    // 1. Open "_common_data/regression/7455/clipboard.gb"
+    GTFileDialog::openFile(os, testDir + "_common_data/regression/7455/clipboard.gb");
+    GTUtilsSequenceView::checkSequenceViewWindowIsActive(os);
+
+    // 2. Open the "Find restriction sites" dialog, choose "AaaI" (vary first one) only and click OK.
+    GTUtilsDialog::waitForDialog(os, new FindEnzymesDialogFiller(os, {"AaaI"}));
+    GTWidget::click(os, GTWidget::findWidget(os, "Find restriction sites_widget"));
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    // 3. Right click -> Cloning -> Digest into fragments...
+    class DigestScenario : public CustomScenario {
+    public:
+        void run(HI::GUITestOpStatus& os) override {
+            // 4. Select "AaaI" and click "Add---->"
+            // 5. Go to the "Conserved annotations" tab
+            QWidget* dialog = GTWidget::getActiveModalWidget(os);
+            QListWidget* availableEnzymeWidget = GTWidget::findExactWidget<QListWidget*>(os, "availableEnzymeWidget", dialog);
```

Comment:
nit: type is typed twice in the same line

Reply:
OK